### ISSUE TITLE
Pain with importing chai 3.5.0 - ReferenceError: Can't find variable: require$$0 - Buffer shim

### ISSUE THREAD

**Issue by rosskevin** [rollup/rollup#614]
**Usage error message**:
```
ReferenceError: Can't find variable: require$$0
```

**Rollup output file and settings here**:
https://gist.github.com/rosskevin/940d06e084f71fc75981196cd93c5fb4

**Rollup message**
```
preferring built-in module 'buffer' over local alternative at '/Users/kross/projects/picker.js/node_modules/buffer/index.js', pass 'preferBuiltins: false' to disable this behavior or 'preferBuiltins: true' to disable this warning
[15:59:45] Treating 'buffer' as external dependency
[15:59:45] No name was provided for external module 'buffer' in options.globals – guessing 'require$$0'
```

**Options**
```javascript
{
    entry: 'test/index.js',
    sourceMap: true,
    format: 'iife',
    plugins: [ 
        commonjs({
            include: 'node_modules/**',
            extensions: [
                '.js'
            ]
        }),
        nodeResolve({
            jsnext: true,
            main: true,
            builtins: false,
            browser: true,
            extensions: [
                '.js',
                '.json'
            ]
        }),
        babel({
            babelrc: false,
            presets: [
                'es2015-rollup'
            ]
        }),
    ],
    dest: 'dist/picker-tests.js.iife.js',
    moduleName: 'pickerTests'
}
```

I tried to add
```javascript
external: [
          'buffer'
        ],
        globals: {
          buffer: 'buffer'
        }
```

But that failed.  The only thing in this file is the rollup output of 

```javascript
import {expect} from 'chai'
```

What am I missing with the [chai (3.5.0)](https://github.com/chaijs/chai/blob/master/lib/chai.js) import or rollup configuration options?

**Comment by rosskevin** [comment]
Perhaps I have the wrong globals/external config - here's the `buffer` reference:
https://github.com/chaijs/chai/blob/772fd5e99b3dd52938feb4811d24e75609a199f6/chai.js#L5614

**Comment by rosskevin** [comment]
My local `buffer` is https://github.com/feross/buffer

**Comment by rosskevin** [comment]
Ok, I've hacked this together...and it feels pretty dirty.  What is a better way?

1. Added to rollup options

    ```javascript
globals: {
    buffer: 'Buffer'
}
    ```
1. Added a script tag to the mocha `testrunner.html`:

    ```html
 <script>
    var Buffer = {}
    Buffer.isBuffer = function() { return false }
 </script>
    ```

**Comment by rosskevin** [comment]
Yeah, so while I tricked it past the error, I still cannot use chai...I'm just missing something.

**Comment by rosskevin** [comment]
Ok, solved it (though I would still love to know why I can't import into an ES2015 class).

**Solution: ** Import the chai standalone in the `testrunner.html` and set a global variable `expect`:

```html
<script src="../node_modules/chai/chai.js"></script>
<script src="../node_modules/chai-jquery/chai-jquery.js"></script>
<script>
  var expect = chai.expect
</script>
```

**Comment by SupremeTechnopriest** [comment]
@rosskevin I just ran into the same issue.  I would like to use the same set of tests for node and browser.  The issue is with chai, they have planned to fix it, but i just tested `4.0.0-canary.1` and it's not working yet.  Heres the issue (although it been closed) https://github.com/chaijs/chai/issues/748
